consensus/src/quorum_store/batch_reader.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_types::transaction::SignedTransaction;
+use consensus_types::proof_of_store::{LogicalTime, ProofOfStore};
+use executor_types::Error;
+use tokio::sync::oneshot;
+
+pub struct BatchReader {}
+
+impl BatchReader {
+    pub async fn get_batch(
+        &self,
+        _proof: ProofOfStore,
+    ) -> oneshot::Receiver<Result<Vec<SignedTransaction>, Error>> {
+        let (_tx, rx) = oneshot::channel();
+
+        // TODO: coming soon
```

Comment:
add a `todo!()`

Reply:
I don't think we want to panic here, just a no-op.

Reply:
We will anyway change it with the next quorum store PRs